pkg/controller/endpoint/endpoints_controller_test.go

```diff
@@ -1625,12 +1625,12 @@ func TestPodUpdatesBatching(t *testing.T) {
 					podIP:   "10.0.0.0",
 				},
 				{
-					delay:   100 * time.Millisecond,
+					delay:   800 * time.Millisecond,
```

Comment:
@yedamao Could you please explain me that why we need to increase delay duration for this PR, why we cannot just migrate endpoints controller tests to mock clock.